tests/run_tests.py

```diff
@@ -1,3 +1,8 @@
+import sys
 import unittest
 
+curdir = sys.path[0]
+pardir = curdir + '/..'
+sys.path.append(pardir)
```

Comment:
Maybe let's keep it the same as I do in specter-diy,
then all imports will be the same as in normal firmware.

```suggestion
sys.path.append('../libs/common')
sys.path.append('../libs/unix')
sys.path.append('../usermods/udisplay_f469/display_unixport')
```

Reply:
Fixed